Utilities/Create-AzureConfigFilesForJenkins.ps1

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+#
+<#
+.SYNOPSIS
+    This file creates 'Property Files' for jenkins dynamic menu creation.
+    These 'Property Files' can be used to create dynamic jenkins menus using parameter type - "Extended Choice Parameter".
+
+.PARAMETER
+	See source code for the detailed parameters
+
+.NOTES
+	PREREQUISITES:
+	1) You should have a valid Azure subscription and read access.
```

Comment:
It needs AzAccount connect, Connect-AzAccount first.

Reply:
This statement is a general statement that user needs to have valid azure subscription.
Authentication is handled in script.